optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/header/MenuView.java

```diff
@@ -101,7 +109,7 @@ public void employeeRoster(final ClickEvent e) {
     }
 
     private void goTo(final Pages.Id pageId,
-                      final ClickEvent event) {
+            final ClickEvent event) {
```

Comment:
Are we using the same code formatter? Maybe the eclipse version does not align the parameters when they're in different lines, can you check that?

Reply:
I honestly would prefer if we had a mvn:format command so we don't need to set the formatters on our editors, but let me check.

Reply:
I only half set up the eclipse formatter; it has its formatter in three parts: cleanup, import order and formatter... why so complicated...